### ISSUE TITLE
[BUG] split_out on std aggregation causes ArrowTypeError

### ISSUE THREAD

**Issue by Nanthini10** [rapidsai/cudf#3480]
**Describe the bug**

Using `split_out` argument with `std` aggregation on a `dask-cudf` dataframe results in the following error

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/column/column.py in as_column(arbitrary, nan_as_null, dtype, name)
   1384             data = as_column(
-> 1385                 memoryview(arbitrary), dtype=dtype, nan_as_null=nan_as_null
   1386             )

TypeError: memoryview: a bytes-like object is required, not 'Index'

During handling of the above exception, another exception occurred:

ArrowTypeError                            Traceback (most recent call last)
/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/column/column.py in as_column(arbitrary, nan_as_null, dtype, name)
   1401                 data = as_column(
-> 1402                     pa.array(arbitrary, type=pa_type, from_pandas=nan_as_null),
   1403                     dtype=dtype,

/opt/conda/envs/rapids/lib/python3.7/site-packages/pyarrow/array.pxi in pyarrow.lib.array()

/opt/conda/envs/rapids/lib/python3.7/site-packages/pyarrow/array.pxi in pyarrow.lib._ndarray_to_array()

/opt/conda/envs/rapids/lib/python3.7/site-packages/pyarrow/error.pxi in pyarrow.lib.check_status()

ArrowTypeError: Expected a bytes object, got a 'tuple' object

During handling of the above exception, another exception occurred:

ArrowTypeError                            Traceback (most recent call last)
<ipython-input-5-f6a973d44b1b> in <module>
      1 std_df = ddf.groupby('a').b.std(split_out=2)
----> 2 std_df.compute()

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/base.py in compute(self, **kwargs)
    163         dask.base.compute
    164         """
--> 165         (result,) = compute(self, traverse=False, **kwargs)
    166         return result
    167 

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/base.py in compute(*args, **kwargs)
    434     keys = [x.__dask_keys__() for x in collections]
    435     postcomputes = [x.__dask_postcompute__() for x in collections]
--> 436     results = schedule(dsk, keys, **kwargs)
    437     return repack([f(r, *a) for r, (f, a) in zip(results, postcomputes)])
    438 

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/local.py in get_sync(dsk, keys, **kwargs)
    525     """
    526     kwargs.pop("num_workers", None)  # if num_workers present, remove it
--> 527     return get_async(apply_sync, 1, dsk, keys, **kwargs)
    528 
    529 

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/local.py in get_async(apply_async, num_workers, dsk, result, cache, get_id, rerun_exceptions_locally, pack_exception, raise_exception, callbacks, dumps, loads, **kwargs)
    469             # Seed initial tasks into the thread pool
    470             while state["ready"] and len(state["running"]) < num_workers:
--> 471                 fire_task()
    472 
    473             # Main loop, wait on tasks to finish, insert new ones

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/local.py in fire_task()
    464                         pack_exception,
    465                     ),
--> 466                     callback=queue.put,
    467                 )
    468 

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/local.py in apply_sync(func, args, kwds, callback)
    514 def apply_sync(func, args=(), kwds={}, callback=None):
    515     """ A naive synchronous version of apply_async """
--> 516     res = func(*args, **kwds)
    517     if callback is not None:
    518         callback(res)

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/local.py in execute_task(key, task_info, dumps, loads, get_id, pack_exception)
    225         failed = False
    226     except BaseException as e:
--> 227         result = pack_exception(e, dumps)
    228         failed = True
    229     return key, result, failed

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/local.py in execute_task(key, task_info, dumps, loads, get_id, pack_exception)
    220     try:
    221         task, data = loads(task_info)
--> 222         result = _execute_task(task, data)
    223         id = get_id()
    224         result = dumps((result, id))

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/core.py in _execute_task(arg, cache, dsk)
    117         func, args = arg[0], arg[1:]
    118         args2 = [_execute_task(a, cache) for a in args]
--> 119         return func(*args2)
    120     elif not ishashable(arg):
    121         return arg

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/dataframe/core.py in hash_shard(df, nparts, split_out_setup, split_out_setup_kwargs)
   4501         h = h.values
   4502     h %= nparts
-> 4503     return {i: df.iloc[h == i] for i in range(nparts)}
   4504 
   4505 

/opt/conda/envs/rapids/lib/python3.7/site-packages/dask/dataframe/core.py in <dictcomp>(.0)
   4501         h = h.values
   4502     h %= nparts
-> 4503     return {i: df.iloc[h == i] for i in range(nparts)}
   4504 
   4505 

/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/indexing.py in __getitem__(self, arg)
    107             if not isinstance(arg, tuple):
    108                 arg = (arg, slice(None))
--> 109             return self._getitem_tuple_arg(arg)
    110 
    111     def __setitem__(self, key, value):

/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/indexing.py in _getitem_tuple_arg(self, arg)
    304                 columns_df._index = self._df._index
    305             else:
--> 306                 columns_df = self._df._columns_view(columns)
    307 
    308         # Iloc Step 2:

/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/dataframe.py in _columns_view(self, columns)
   3912         Return a subset of the DataFrame's columns as a view.
   3913         """
-> 3914         columns = as_index(columns)
   3915         result_columns = OrderedDict({})
   3916         for col in columns:

/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/index.py in as_index(arbitrary, **kwargs)
   1009         )
   1010     else:
-> 1011         return as_index(column.as_column(arbitrary), **kwargs)
   1012 
   1013 

/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/column/column.py in as_column(arbitrary, nan_as_null, dtype, name)
   1414                     data = as_column(
   1415                         np.array(arbitrary, dtype=np_type),
-> 1416                         nan_as_null=nan_as_null,
   1417                     )
   1418     if hasattr(data, "name") and (name is not None):

/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/column/column.py in as_column(arbitrary, nan_as_null, dtype, name)
   1224             data = datetime.DatetimeColumn.from_numpy(arbitrary)
   1225         elif arbitrary.dtype.kind in ("O", "U"):
-> 1226             data = as_column(pa.Array.from_pandas(arbitrary))
   1227         else:
   1228             data = as_column(rmm.to_device(arbitrary), nan_as_null=nan_as_null)

/opt/conda/envs/rapids/lib/python3.7/site-packages/pyarrow/array.pxi in pyarrow.lib.Array.from_pandas()

/opt/conda/envs/rapids/lib/python3.7/site-packages/pyarrow/array.pxi in pyarrow.lib.array()

/opt/conda/envs/rapids/lib/python3.7/site-packages/pyarrow/array.pxi in pyarrow.lib._ndarray_to_array()

/opt/conda/envs/rapids/lib/python3.7/site-packages/pyarrow/error.pxi in pyarrow.lib.check_status()

ArrowTypeError: Expected a bytes object, got a 'tuple' object
```

**Steps/Code to reproduce bug**
```
import cudf 
import dask_cudf
df = cudf.DataFrame({'a': [1,2,2], 'b': [20,305,91]})
ddf = dask_cudf.from_cudf(df, npartitions=2)
std_df = ddf.groupby('a').b.std(split_out=2)
std_df.compute() # Results in above error
```

**Expected behavior**
The function is expected to return the standard deviation. The code below demonstrates same behavior with `mean` aggregation.

```
mean_df = ddf.groupby('a').b.mean(split_out=2)
mean_df.compute()
a
1     20.0
2    198.0
Name: b, dtype: float64
```

Also, worth noting that without the `split_out` argument `std` works fine.
```
std_df = ddf.groupby('a').b.std()
std_df.compute() # This works without a problem

a
1           null
2    151.3208512
Name: b, dtype: float64
```
**Environment overview (please complete the following information)**
 - Environment location: Docker
 - Method of cuDF install: From source (0.11)

**Environment details**
```
    
     **git***
     commit c021a74ca84a6157ea2b6b920ee9692676809ebc (HEAD -> branch-0.11, origin/branch-0.11, origin/HEAD)
     Merge: 5cde46d d7f8cf1
     Author: Keith Kraus <kkraus@nvidia.com>
     Date:   Mon Nov 25 19:17:21 2019 -0500
     
     Merge pull request #3442 from VibhuJawa/fea-set-item-dataframe-bool-list-index
     
     [REVIEW] Add Bool-index + Multi column + DataFrame support for set-item
     **git submodules***
     b165e1fb11eeea64ccf95053e40f2424312599cc thirdparty/cub (v1.7.1)
     63f644be44201467e3938d59ed9d89cc8725c35d thirdparty/jitify (remotes/origin/feature/api_v2_v0.10)
     39125e0e476b960c2001f1ec76a3441335ff91b2 thirdparty/libcudacxx (0.8.1-94-g39125e0)
     08bc464bd8f4d779e4294305aa7dadebcebcc507 thirdparty/libcudacxx/libcxx (heads/rapidsai-interop)
     
     ***OS Information***
     DISTRIB_ID=Ubuntu
     DISTRIB_RELEASE=16.04
     DISTRIB_CODENAME=xenial
     DISTRIB_DESCRIPTION="Ubuntu 16.04.6 LTS"
     NAME="Ubuntu"
     VERSION="16.04.6 LTS (Xenial Xerus)"
     ID=ubuntu
     ID_LIKE=debian
     PRETTY_NAME="Ubuntu 16.04.6 LTS"
     VERSION_ID="16.04"
     VERSION_CODENAME=xenial
     UBUNTU_CODENAME=xenial
     Linux bbdc36128080 4.4.0-154-generic #181-Ubuntu SMP Tue Jun 25 05:29:03 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
     
     ***GPU Information***
     Wed Nov 27 18:43:31 2019
     +-----------------------------------------------------------------------------+
     | NVIDIA-SMI 410.104      Driver Version: 410.104      CUDA Version: 10.0     |
     |-------------------------------+----------------------+----------------------+
     | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
     | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
     |===============================+======================+======================|
     |   0  Tesla T4            Off  | 00000000:3B:00.0 Off |                    0 |
     | N/A   52C    P0    29W /  70W |   1132MiB / 15079MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   1  Tesla T4            Off  | 00000000:5E:00.0 Off |                    0 |
     | N/A   59C    P0    28W /  70W |    289MiB / 15079MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   2  Tesla T4            Off  | 00000000:AF:00.0 Off |                    0 |
     | N/A   50C    P0    29W /  70W |    289MiB / 15079MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   3  Tesla T4            Off  | 00000000:D8:00.0 Off |                    0 |
     | N/A   49C    P0    29W /  70W |    289MiB / 15079MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     
     +-----------------------------------------------------------------------------+
     | Processes:                                                       GPU Memory |
     |  GPU       PID   Type   Process name                             Usage      |
     |=============================================================================|
     +-----------------------------------------------------------------------------+
     
     ***CPU***
     Architecture:          x86_64
     CPU op-mode(s):        32-bit, 64-bit
     Byte Order:            Little Endian
     CPU(s):                64
     On-line CPU(s) list:   0-63
     Thread(s) per core:    2
     Core(s) per socket:    16
     Socket(s):             2
     NUMA node(s):          2
     Vendor ID:             GenuineIntel
     CPU family:            6
     Model:                 85
     Model name:            Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz
     Stepping:              4
     CPU MHz:               2095.092
     BogoMIPS:              4191.35
     Virtualization:        VT-x
     L1d cache:             32K
     L1i cache:             32K
     L2 cache:              1024K
     L3 cache:              22528K
     NUMA node0 CPU(s):     0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48,50,52,54,56,58,60,62
     NUMA node1 CPU(s):     1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43,45,47,49,51,53,55,57,59,61,63
     
     ***CMake***
     /opt/conda/envs/rapids/bin/cmake
     cmake version 3.15.5
     
     CMake suite maintained and supported by Kitware (kitware.com/cmake).
     
     ***g++***
     /usr/bin/g++
     g++ (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609
     Copyright (C) 2015 Free Software Foundation, Inc.
     This is free software; see the source for copying conditions.  There is NO
     warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
     
     
     ***nvcc***
     /usr/local/cuda/bin/nvcc
     nvcc: NVIDIA (R) Cuda compiler driver
     Copyright (c) 2005-2018 NVIDIA Corporation
     Built on Sat_Aug_25_21:08:01_CDT_2018
     Cuda compilation tools, release 10.0, V10.0.130
     
     ***Python***
     /opt/conda/envs/rapids/bin/python
     Python 3.7.3
     
     ***Environment Variables***
     PATH                            : /opt/conda/envs/rapids/bin:/opt/conda/condabin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/conda/bin:/conda/bin:/conda/bin:/usr/lib/jvm/java-8-openjdk-amd64/bin:/usr/share/sbt-launcher-packaging/bin/sbt-launch.jar/bin:/usr/lib/spark/bin:/usr/lib/spark/sbin
     LD_LIBRARY_PATH                 : /opt/conda/envs/rapids/lib:/usr/local/nvidia/lib:/usr/local/nvidia/lib64:/usr/local/cuda/lib64:/usr/local/lib
     NUMBAPRO_NVVM                   :
     NUMBAPRO_LIBDEVICE              :
     CONDA_PREFIX                    : /opt/conda/envs/rapids
     PYTHON_PATH                     :
     
     ***conda packages***
     /opt/conda/condabin/conda

     # packages in environment at /opt/conda/envs/rapids:
     #
     # Name                    Version                   Build  Channel
     _libgcc_mutex             0.1                        main
     _tflow_select             2.1.0                       gpu
     absl-py                   0.8.1                    py37_0    conda-forge
     aiohttp                   3.6.1            py37h516909a_0    conda-forge
     alabaster                 0.7.12                     py_0    conda-forge
     appdirs                   1.4.3                      py_1    conda-forge
     arrow-cpp                 0.15.0           py37h090bef1_2    conda-forge
     aspy.yaml                 1.3.0                      py_0    conda-forge
     astor                     0.7.1                      py_0    conda-forge
     async-timeout             3.0.1                   py_1000    conda-forge
     attrs                     19.3.0                     py_0    conda-forge
     babel                     2.7.0                      py_0    conda-forge
     backcall                  0.1.0                      py_0    conda-forge
     black                     19.10b0                  py37_0    conda-forge
     blas                      2.14                   openblas    conda-forge
     bleach                    3.1.0                      py_0    conda-forge
     bokeh                     1.4.0                    py37_0    conda-forge
     boost-cpp                 1.70.0               h8e57a91_2    conda-forge
     brotli                    1.0.7             he1b5a44_1000    conda-forge
     bzip2                     1.0.8                h516909a_1    conda-forge
     c-ares                    1.15.0            h516909a_1001    conda-forge
     ca-certificates           2019.9.11            hecc5488_0    conda-forge
     cached-property           1.5.1                      py_0    conda-forge
     cairo                     1.16.0            hfb77d84_1002    conda-forge
     certifi                   2019.9.11                py37_0    conda-forge
     cffi                      1.13.2           py37h8022711_0    conda-forge
     cfgv                      2.0.1                      py_0    conda-forge
     cfitsio                   3.470                hb60a0a2_2    conda-forge
     chardet                   3.0.4                 py37_1003    conda-forge
     click                     7.0                      pypi_0    pypi
     cloudpickle               1.2.2                      py_1    conda-forge
     cmake                     3.15.5               hf94ab9c_0    conda-forge
     cmake_setuptools          0.1.3                      py_0    rapidsai-nightly
     commonmark                0.9.1                      py_0    conda-forge
     cryptography              2.8              py37h72c5cf5_0    conda-forge
     cudatoolkit               10.0.130                      0    nvidia
     cudf                      0.11.0a0+3458.gc021a74          pypi_0    pypi
     cudnn                     7.6.0                cuda10.0_0    nvidia
     cupti                     10.0.130                      0
     cupy                      6.3.0            py37h88562e5_0    rapidsai
     curl                      7.65.3               hf8cf82a_0    conda-forge
     cycler                    0.10.0                     py_2    conda-forge
     cython                    0.29.14          py37he1b5a44_0    conda-forge
     cytoolz                   0.10.1           py37h516909a_0    conda-forge
     dask                      2.8.1+4.g3960c65          pypi_0    pypi
     dask-cudf                 0.11.0a0+3458.gc021a74          pypi_0    pypi
     dask-glm                  0.2.0                      py_1    conda-forge
     dask-labextension         1.0.3                      py_0    conda-forge
     dask-ml                   1.1.1                      py_0    conda-forge
     distributed               2.8.1+6.g6af2617          pypi_0    pypi
     dlpack                    0.2                  he1b5a44_1    conda-forge
     docutils                  0.15.2                   py37_0    conda-forge
     double-conversion         3.1.5                he1b5a44_2    conda-forge
     doxygen                   1.8.16               hd1b7508_1    conda-forge
     editdistance              0.5.3            py37he1b5a44_0    conda-forge
     entrypoints               0.3                   py37_1000    conda-forge
     expat                     2.2.5             he1b5a44_1004    conda-forge
     fastavro                  0.22.7           py37h516909a_0    conda-forge
     fastrlock                 0.4             py37he1b5a44_1000    conda-forge
     flake8                    3.7.9                    py37_0    conda-forge
     flatbuffers               1.11.0               he1b5a44_0    conda-forge
     fontconfig                2.13.1            h86ecdb6_1001    conda-forge
     freetype                  2.10.0               he983fc9_1    conda-forge
     freexl                    1.0.5             h14c3975_1002    conda-forge
     fribidi                   1.0.5             h516909a_1002    conda-forge
     fsspec                    0.6.0                      py_0    conda-forge
     future                    0.18.2                   py37_0    conda-forge
     geotiff                   1.5.1                hbd99317_7    conda-forge
     gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
     gflags                    2.2.2             he1b5a44_1002    conda-forge
     giflib                    5.1.7                h516909a_1    conda-forge
     glib                      2.58.3          py37h6f030ca_1002    conda-forge
     glog                      0.4.0                he1b5a44_1    conda-forge
     gmp                       6.1.2             hf484d3e_1000    conda-forge
     google-pasta              0.1.8                      py_0    conda-forge
     graphite2                 1.3.13            hf484d3e_1000    conda-forge
     graphviz                  2.40.1               h0511662_2    conda-forge
     grpc-cpp                  1.23.0               h18db393_0    conda-forge
     grpcio                    1.23.0           py37he9ae1f9_0    conda-forge
     gst-plugins-base          1.14.5               h0935bb2_0    conda-forge
     gstreamer                 1.14.5               h36ae1b5_0    conda-forge
     h5py                      2.10.0          nompi_py37h513d04c_100    conda-forge
     harfbuzz                  2.4.0                h9f30f68_3    conda-forge
     hdf4                      4.2.13            hf30be14_1003    conda-forge
     hdf5                      1.10.5          nompi_h3c11f04_1104    conda-forge
     heapdict                  1.0.1                      py_0    conda-forge
     hypothesis                4.47.2                   py37_0    conda-forge
     icu                       64.2                 he1b5a44_1    conda-forge
     identify                  1.4.7                      py_0    conda-forge
     idna                      2.8                   py37_1000    conda-forge
     imagesize                 1.1.0                      py_0    conda-forge
     importlib_metadata        0.23                     py37_0    conda-forge
     ipykernel                 5.1.3            py37h5ca1d4c_0    conda-forge
     ipython                   7.9.0            py37h5ca1d4c_1    conda-forge
     ipython_genutils          0.2.0                      py_1    conda-forge
     isort                     4.3.21                   py37_0    conda-forge
     jedi                      0.15.1                   py37_0    conda-forge
     jinja2                    2.10.3                     py_0    conda-forge
     joblib                    0.14.0                     py_0    conda-forge
     jpeg                      9c                h14c3975_1001    conda-forge
     json-c                    0.13.1            h14c3975_1001    conda-forge
     json5                     0.8.5                      py_0    conda-forge
     jsonschema                3.2.0                    py37_0    conda-forge
     jupyter-server-proxy      1.2.0                      py_0    conda-forge
     jupyter_client            5.3.3                    py37_1    conda-forge
     jupyter_core              4.6.1                    py37_0    conda-forge
     jupyterlab                1.0.7                    py37_0    conda-forge
     jupyterlab_server         1.0.6                      py_0    conda-forge
     kealib                    1.4.10            h58c409b_1005    conda-forge
     keras-applications        1.0.8                      py_1    conda-forge
     keras-preprocessing       1.1.0                      py_0    conda-forge
     kiwisolver                1.1.0            py37hc9558a2_0    conda-forge
     krb5                      1.16.3            h05b26f9_1001    conda-forge
     lapack                    3.6.1                ha44fe06_2    conda-forge
     libblas                   3.8.0               14_openblas    conda-forge
     libcblas                  3.8.0               14_openblas    conda-forge
     libclang                  8.0.0                h6bb024c_0    rapidsai
     libcurl                   7.65.3               hda55be3_0    conda-forge
     libcypher-parser          0.6.2                         1    rapidsai
     libdap4                   3.20.4               hd3bb157_0    conda-forge
     libedit                   3.1.20170329      hf8c457e_1001    conda-forge
     libevent                  2.1.10               h72c5cf5_0    conda-forge
     libffi                    3.2.1             he1b5a44_1006    conda-forge
     libgcc-ng                 7.3.0                hdf63c60_0
     libgdal                   2.4.3                h2f07a13_9    conda-forge
     libgfortran               3.0.0                         1    conda-forge
     libgfortran-ng            7.3.0                hdf63c60_2    conda-forge
     libiconv                  1.15              h516909a_1005    conda-forge
     libkml                    1.3.0             h4fcabce_1010    conda-forge
     liblapack                 3.8.0               14_openblas    conda-forge
     liblapacke                3.8.0               14_openblas    conda-forge
     libllvm8                  8.0.1                hc9558a2_0    conda-forge
     libnetcdf                 4.7.1           nompi_h94020b1_102    conda-forge
     libopenblas               0.3.7                h6e990d7_3    conda-forge
     libprotobuf               3.8.0                h8b12597_0    conda-forge
     librmm                    0.11.0b191126       cuda10.0_55    rapidsai-nightly
     libsodium                 1.0.17               h516909a_0    conda-forge
     libspatialite             4.3.0a            h4f6d029_1032    conda-forge
     libssh2                   1.8.2                h22169c7_2    conda-forge
     libstdcxx-ng              7.3.0                hdf63c60_0
     libtiff                   4.1.0                hfc65ed5_0    conda-forge
     libtool                   2.4.6             h14c3975_1002    conda-forge
     libuuid                   2.32.1            h14c3975_1000    conda-forge
     libuv                     1.33.1               h516909a_0    conda-forge
     libxcb                    1.13              h14c3975_1002    conda-forge
     libxml2                   2.9.10               hee79883_0    conda-forge
     llvmlite                  0.30.0           py37h8b12597_1    conda-forge
     locket                    0.2.0                      py_2    conda-forge
     lz4-c                     1.8.3             he1b5a44_1001    conda-forge
     make                      4.2.1             h14c3975_2004    conda-forge
     markdown                  3.0.1                    pypi_0    pypi
     markupsafe                1.1.1            py37h516909a_0    conda-forge
     matplotlib                3.1.2                    py37_1    conda-forge
     matplotlib-base           3.1.2            py37h250f245_1    conda-forge
     mccabe                    0.6.1                      py_1    conda-forge
     mistune                   0.8.4           py37h516909a_1000    conda-forge
     more-itertools            7.2.0                      py_0    conda-forge
     msgpack-python            0.6.2            py37hc9558a2_0    conda-forge
     multidict                 4.5.2           py37h14c3975_1000    conda-forge
     multipledispatch          0.6.0                      py_0    conda-forge
     mypy_extensions           0.4.3                    py37_0    conda-forge
     nbconvert                 5.6.1                    py37_0    conda-forge
     nbformat                  4.4.0                      py_1    conda-forge
     nbsphinx                  0.5.0                      py_0    conda-forge
     nccl                      2.4.6.1              cuda10.0_0    nvidia
     ncurses                   6.1               hf484d3e_1002    conda-forge
     networkx                  2.4                        py_0    conda-forge
     nodeenv                   1.3.3                      py_0    conda-forge
     notebook                  6.0.1                    py37_0    conda-forge
     numba                     0.46.0           py37hb3f55d8_1    conda-forge
     numpy                     1.16.5           py37h95a1406_0    conda-forge
     numpydoc                  0.9.1                      py_0    conda-forge
     nvstrings-cuda100         0.0.0.dev0               pypi_0    pypi
     olefile                   0.46                       py_0    conda-forge
     openjpeg                  2.3.1                h981e76c_2    conda-forge
     openssl                   1.1.1d               h516909a_0    conda-forge
     packaging                 19.2                       py_0    conda-forge
     pandas                    0.24.2           py37hb3f55d8_1    conda-forge
     pandoc                    1.19.2                        0    conda-forge
     pandocfilters             1.4.2                      py_1    conda-forge
     pango                     1.42.4               ha030887_1    conda-forge
     parquet-cpp               1.5.1                         2    conda-forge
     parso                     0.5.1                      py_0    conda-forge
     partd                     1.0.0                      py_0    conda-forge
     pathspec                  0.6.0                      py_0    conda-forge
     patsy                     0.5.1                      py_0    conda-forge
     pcre                      8.43                 he1b5a44_0    conda-forge
     pexpect                   4.7.0                    py37_0    conda-forge
     pickleshare               0.7.5                 py37_1000    conda-forge
     pillow                    6.2.1            py37hd70f55b_1    conda-forge
     pip                       19.3.1                   py37_0    conda-forge
     pixman                    0.38.0            h516909a_1003    conda-forge
     pluggy                    0.13.0                   py37_0    conda-forge
     poppler                   0.67.0               h14e79db_8    conda-forge
     poppler-data              0.4.9                         1    conda-forge
     postgresql                11.5                 hc63931a_2    conda-forge
     pre_commit                1.18.1                   py37_0    conda-forge
     proj                      6.2.1                hc80f0dc_0    conda-forge
     prometheus_client         0.7.1                      py_0    conda-forge
     prompt_toolkit            2.0.10                     py_0    conda-forge
     protobuf                  3.8.0            py37he1b5a44_2    conda-forge
     psutil                    5.6.7            py37h516909a_0    conda-forge
     pthread-stubs             0.4               h14c3975_1001    conda-forge
     ptyprocess                0.6.0                   py_1001    conda-forge
     py                        1.8.0                      py_0    conda-forge
     pyarrow                   0.15.0           py37h8b68381_1    conda-forge
     pycodestyle               2.5.0                      py_0    conda-forge
     pycparser                 2.19                     py37_1    conda-forge
     pyflakes                  2.1.1                      py_0    conda-forge
     pygments                  2.4.2                      py_0    conda-forge
     pynvml                    8.0.3                    pypi_0    pypi
     pyopenssl                 19.0.0                   py37_0    conda-forge
     pyparsing                 2.4.5                      py_0    conda-forge
     pyqt                      5.9.2            py37hcca6a23_4    conda-forge
     pyrsistent                0.15.6           py37h516909a_0    conda-forge
     pysocks                   1.7.1                    py37_0    conda-forge
     pytest                    5.3.1                    py37_0    conda-forge
     python                    3.7.3                h33d41f4_1    conda-forge
     python-dateutil           2.8.1                      py_0    conda-forge
     pytz                      2019.3                     py_0    conda-forge
     pyyaml                    5.1.2            py37h516909a_0    conda-forge
     pyzmq                     18.1.1           py37h1768529_0    conda-forge
     qt                        5.9.7                h0c104cb_3    conda-forge
     rapidjson                 1.1.0             he1b5a44_1002    conda-forge
     re2                       2019.11.01           he1b5a44_0    conda-forge
     readline                  8.0                  hf8c457e_0    conda-forge
     recommonmark              0.6.0                      py_0    conda-forge
     regex                     2019.11.1        py37h516909a_0    conda-forge
     requests                  2.22.0                   py37_1    conda-forge
     rhash                     1.3.6             h14c3975_1001    conda-forge
     rmm                       0.11.0b191126           py37_55    rapidsai-nightly
     scikit-learn              0.21.3           py37hcdab131_0    conda-forge
     scipy                     1.3.2            py37h921218d_0    conda-forge
     seaborn                   0.9.0                      py_2    conda-forge
     send2trash                1.5.0                      py_0    conda-forge
     setuptools                42.0.1                   py37_0    conda-forge
     simpervisor               0.3                        py_1    conda-forge
     sip                       4.19.8          py37hf484d3e_1000    conda-forge
     six                       1.13.0                   py37_0    conda-forge
     snappy                    1.1.7             he1b5a44_1002    conda-forge
     snowballstemmer           2.0.0                      py_0    conda-forge
     sortedcontainers          2.1.0                      py_0    conda-forge
     sqlite                    3.30.1               hcee41ef_0    conda-forge
     statsmodels               0.10.2           py37hc1659b7_0    conda-forge
     streamz                   0.5.2                    pypi_0    pypi
     tblib                     1.4.0                      py_0    conda-forge
     tensorboard               1.14.0                   py37_0    conda-forge
     tensorflow                1.14.0          gpu_py37h4491b45_0
     tensorflow-base           1.14.0          gpu_py37h8d69cac_0
     tensorflow-estimator      1.14.0           py37h5ca1d4c_0    conda-forge
     termcolor                 1.1.0                      py_2    conda-forge
     terminado                 0.8.3                    py37_0    conda-forge
     testpath                  0.4.4                      py_0    conda-forge
     thrift-cpp                0.12.0            hf3afdfd_1004    conda-forge
     tk                        8.6.10               hed695b0_0    conda-forge
     toml                      0.10.0                     py_0    conda-forge
     toolz                     0.10.0                     py_0    conda-forge
     tornado                   6.0.3            py37h516909a_0    conda-forge
     traitlets                 4.3.3                    py37_0    conda-forge
     typed-ast                 1.4.0            py37h516909a_0    conda-forge
     typing_extensions         3.7.4.1                  py37_0    conda-forge
     tzcode                    2019a             h516909a_1002    conda-forge
     umap-learn                0.3.10                   py37_0    conda-forge
     uriparser                 0.9.3                he1b5a44_1    conda-forge
     urllib3                   1.25.7                   py37_0    conda-forge
     virtualenv                16.7.5                     py_0    conda-forge
     wcwidth                   0.1.7                      py_1    conda-forge
     werkzeug                  0.16.0                     py_0    conda-forge
     wheel                     0.33.6                   py37_0    conda-forge
     wrapt                     1.11.2           py37h516909a_0    conda-forge
     xerces-c                  3.2.2             h8412b87_1004    conda-forge
     xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
```

**Comment by VibhuJawa** [comment]
CC: @rjzamora , @beckernick

**Comment by beckernick** [comment]
A quick debug indicates that this is likely the issue:

```python
> /opt/conda/envs/rapids/lib/python3.6/site-packages/dask/dataframe/core.py(4495)<dictcomp>()
   4493         h = h.values
   4494     h %= nparts
-> 4495     return {i: df.iloc[h == i] for i in range(nparts)}
   4496 
   4497 

ipdb>  df
     b  (b, -x2)  (b, -count)
a                            
1   20       400            1
2  396    101306            2
ipdb>  type(df)
<class 'cudf.core.dataframe.DataFrame'>
ipdb>  df.columns
Index(['b', ('b', '-x2'), ('b', '-count')], dtype='object')
```

Once we have this dataframe with this column names `Index`, we then can't use `as_column` as pyarrow doesn't know what to do with this object that contains a tuple internally.

```python
ipdb>  type(columns)
<class 'pandas.core.indexes.base.Index'>
ipdb>  arbitrary
Index(['b', ('b', '-x2'), ('b', '-count')], dtype='object')
ipdb>  down
> /opt/conda/envs/rapids/lib/python3.6/site-packages/cudf/core/column/column.py(1226)as_column()
   1224             data = datetime.DatetimeColumn.from_numpy(arbitrary)
   1225         elif arbitrary.dtype.kind in ("O", "U"):
-> 1226             data = as_column(pa.Array.from_pandas(arbitrary))
```

**Comment by beckernick** [comment]
The underlying issue causing this is MultiIndex related. Filing a separate issue and cross-linking

**Comment by beckernick** [comment]
This has been resolved. Closing.
